Socket.php

```diff
@@ -49,21 +49,201 @@
 class Socket extends HoaSocket
 {
     /**
+     * Entity type: isuser.
+     *
+     * @const string
+     */
+    const ENTTYPE_ISUSER = 'isuser';
+
+    /**
+     * Entity type: ischannel.
+     *
+     * @const string
+     */
+    const ENTTYPE_ISCHANNEL = 'ischannel';
```

Comment:
I think you can use integers instead strings here. It will be lighter. And since we have constants, it does not matter.